### ISSUE TITLE
Can only use Search().series() once

### ISSUE THREAD

**Issue by kwodzicki** [phate89/tvdbsimple#14]
It looks like in the Search().series() method, the series method is overwritten by the response from the request, but the response is also returned. This does not make much sense because each Search() instance can only be used once.

**Comment by rmartin16** [comment]
yeah...true story. i tried to outline most of the issues i found for this package in #7. some of the issues were fixed but other require refactoring or breaks in backwards compatibility.

**Comment by phate89** [comment]
The problem is that I will break compatibility if i drop this behavior. I did a stupid change from original celiao not sure why but now I can't fix it without breaking it.. I'm considering having a new method that behaves like this method should.

**Comment by Jonny007-MKD** [comment]
I, too, stumbled over this :(
